src/client/views/cube-view/cube-view.tsx

```diff
@@ -229,11 +231,13 @@ export class CubeView extends React.Component<CubeViewProps, CubeViewState> {
     this.setState({ updatingMaxTime: true });
 
     DataCube.queryMaxTime(dataCube)
-      .then(updatedMaxTime => {
+      .then(maxTime => {
         if (!this.mounted) return;
+        if (!dataCube.refreshRule.isRealtime() && maxTime.getTime() === timekeeper.getTime(dataCube.name).getTime()) return;
```

Comment:
Ugly calls chain, why not maxTime === timekeeper.getTime(dataCube.name)? Alternatively I would add method to the timekeeper for time comparison.